web/src/utils/formatting.ts

```diff
@@ -64,21 +73,35 @@ const scalePower = function (maxPower: number | undefined) {
 
   if (value < 1) {
```

Comment:
Do we always want to use EnergyUnits or do we want to use PowerUnits for hourly and energy for the rest. For hourly they have the same value and meaning.

Reply:
I think it makes most sense to stick with energy for consistency :)